### PR TITLE
Delete Currently active profile

### LINKED ISSUE #43: Remove Active Profile when a Profile is deleted
**Describe the bug**
When a profile is active, and the user deletes it by swiping or deletes all profiles, still the currently active profile card remains present.

**To Reproduce**
1. Wait for a profile to be active
2. When it is active, delete it
3. Observe that currently active profile remains as it is.

**Expected behavior**
When all profiles are deleted then no profile is active, so it shouldn't be shown as active

**Screenshots**

**Would you like to work on the issue?**

- [ ] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by coder2699**
Fixes #43

### CODE REVIEW

**Review by rob729 — COMMENTED** [review]
Briefly tell why have you used "ActiveProfileId" in shared preference

**Comment by coder2699** [comment]
> Briefly tell why have you used "ActiveProfileId" in shared preference

When user deletes by swiping a profile, we must check that is it active or not. We could have used ActiveProfileName also to compare that. But considering that there's no restrictions on creating multiple profiles with same name I used ActiveProfileId

File: app/src/main/java/com/rob729/quiethours/util/StartAlarm.kt [hunk 1]
```diff
@@ -51,6 +51,8 @@ class StartAlarm(appContext: Context, workerParams: WorkerParameters) :
         notificationManager.notify(1112, notification)
         StoreSession.writeInt(AppConstants.BEGIN_STATUS, StoreSession.readInt(AppConstants.BEGIN_STATUS) + 1)
         StoreSession.writeString(AppConstants.ACTIVE_PROFILE_NAME, profileName)
+        StoreSession.writeLong(AppConstants.ACTIVE_PROFILE_ID, activeProdileId)
```
Comment (rob729) [inline app/src/main/java/com/rob729/quiethours/util/StartAlarm.kt]: activeProdileId??
Correct the variable name.

**Review by rob729 — CHANGES_REQUESTED** [review]
App not working as expected in case of delete all profiles.
You should thoroughly test the app before raising the PR. It wastes a lot of time while reviewing the code and introduces bugs in some cases when an issue in a PR goes unnoticed and it gets merged
The above is a generalized comment based on your previous and current PR.

**Comment by coder2699** [comment]
> App not working as expected in case of delete all profiles.
> You should thoroughly test the app before raising the PR. It wastes a lot of time while reviewing the code and introduces bugs in some cases when an issue in a PR goes unnoticed and it gets merged
> The above is a generalized comment based on your previous and current PR.

@rob729 I will check once more, regarding delete all profiles and try to improve in future.

File: gradle.properties [hunk 2]
```diff
@@ -6,7 +6,7 @@
 # http://www.gradle.org/docs/current/userguide/build_environment.html
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-org.gradle.jvmargs=-Xmx1536m
+org.gradle.jvmargs=-Xmx1024m
```
Comment (rob729) [inline gradle.properties]: unexpected addition. Remove it.

Comment (coder2699) [reply gradle.properties]: @rob729 this PR was made before I read your  comment regarding delete all profiles. Please don't waste time over this. I'll comment when I resolve all the bugs and push the code

**Comment by coder2699** [comment]
@rob729 Whenever you are free, please let me know if still there is some problem.

**Review by rob729 — APPROVED** [review]

**Comment by rob729** [comment]
give @coder2699 5 points

**Pull request merged**